abbyy_to_epub3/create_epub.py

```diff
@@ -128,7 +130,10 @@ def __init__(
 
         # Initialize all the book's variables cleanly
         self.debug = debug
-        self.args = args
+        self.epubcheck = epubcheck or (
+            # If no epubcheck specified and we're in debug mode, run
+            # --epubcheck warning
+            self.DEFAULT_EPUBCHECK_LEVEL if self.debug and not epubcheck else None)
```

Comment:
SUGGESTION: With short-circuit Boolean operators, I think `epubcheck` is already guaranteed to be false here. Remove from condition?